src/components/Register.tsx

```diff
@@ -0,0 +1,91 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { icon } from "@fortawesome/fontawesome-svg-core/import.macro";
+import Header from "./Header";
+import { useToggle } from "../ts/ToggleHeader";
+import { Link } from "react-router-dom";
+
+export const Login = () => {
+  const lightMode = useToggle((state) => state.light);
+
+  return (
+    <div
+      className={`"App" ${
+        lightMode ? "bg-card-light-blue" : "dark bg-card-dark-blue"
+      } min-h-screen flex flex-col`}
+    >
+      <Header />
+      <div className="relative flex flex-grow">
+        <div className="flex flex-col items-center justify-center flex-1">
+          <button
+            className={`${
+              lightMode
+                ? "bg-card-sky-blue border-black hover:bg-electric-blue"
+                : "bg-black text-white hover:bg-charcoal-black"
+            } border-[3px] rounded-[15px] border-radius: 15px w-[720px] h-[120px] py-2 px-12 mb-24 flex items-center justify-center`}
+          >
+            <FontAwesomeIcon
+              icon={icon({ name: "github", style: "brands" })}
+              className="mr-2 h-12"
+            />
+            <span className="flex-grow text-base">
+              <p
+                className={`${
+                  lightMode ? "text-duke-blue" : "text-white"
+                } text-[48px] ml-2`}
+              >
+                Sign Up with Github
+              </p>
+            </span>
+          </button>
+
+          <button
+            className={`${
+              lightMode
+                ? "bg-card-sky-blue border-black hover:bg-electric-blue"
+                : "bg-black text-white hover:bg-charcoal-black"
+            } border-[3px] rounded-[15px] w-[720px] h-[120px] py-2 px-12 mb-12 flex items-center justify-center`}
+          >
+            <FontAwesomeIcon
+              icon={icon({ name: "google", style: "brands" })}
```

Comment:
@alinz22 The style does not match Google's style.
![image](https://github.com/luminosityleds/client/assets/108915844/6acb32b6-fe0a-4f7a-8b76-e5a6db683a90)

Reply:
@alinz22 It should look like this 
![image](https://github.com/luminosityleds/client/assets/108915844/faf15beb-40ea-4686-9d83-3f0000df385a)

Reply:
@pogi7 We pulled this from the login component in master. We'll need to change that as well, I'll add a bug ticket. @alinz22 see if you can find a way to get this google logo. I know that @Jihnboii was having trouble getting this to work and the black icon was the compromise.

Reply:
@pogi7 @alinz22 I'm going make this a bug ticket, so we can ignore this issue for this PR.